libzfs_core/_libzfs_core.py

```diff
@@ -655,6 +660,276 @@ def lzc_exists(name):
     return bool(ret)
 
 
+def _uncommitted(check_func = None):
```

Comment:
docstring for this and the resulting decorator function would definitely be good